lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,70 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
+      class UnknownOwner < StandardError; end
+      class CannotPostComment < StandardError; end
+
+      extend Forwardable
+
+      attr_reader :config, :repo, :pr_id
+
+      delegate [
+        :github_access_token,
+        :github_comment_template,
+        :github_comment,
+        :github_request_review
+      ] => :config
+
+      def initialize(config, repo, pr_id)
+        @config = config
+        @repo = repo
+        @pr_id = pr_id
+      end
+
+      def owner
+        pull_request.user.login
+      rescue Octokit::Error
+        raise UnknownOwner
+      end
+
+      def assign(reviewer)
+        request_review(reviewer) if github_request_review
+        add_comment(reviewer) if github_comment
+
+        raise CannotPostComment if !github_request_review && !github_comment
```

Comment:
If I'm understanding this correctly, it raises an error if both configuration options are set to false. Is that right? If that's so, should it be the same error as when something goes wrong in the github api? Also it seems like it would be possible to detect this particular problem earlier, maybe even in the constructor. Does that make sense?